src/webview/js/js.js

```diff
@@ -7,6 +7,10 @@ import type {
   MessageInputTyping,
 } from '../webViewHandleUpdates';
 
+// Converts an array-like object to an actual array
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects
+const arrayFrom = arrayLike => Array.prototype.slice.call(arrayLike);
```

Comment:
The comment could mention that Array.from won't necessarily be supported in the WebView, which is why this function exists in the first place.

It's not too much extra effort to look at the commit body, but I think that's the other important detail here, and it would be more visible in the comment.